src/dotnet/APIView/ClientSPA/src/app/_components/review-page/review-page.component.ts

```diff
@@ -211,11 +215,26 @@ export class ReviewPageComponent implements OnInit {
             if (this.diffApiRevisionId) {
               this.diffAPIRevision = this.apiRevisions.filter(x => x.id === this.diffApiRevisionId)[0];
             }
+            this.apiRevisionType = this.activeAPIRevision.apiRevisionType;
+            if (this.activeAPIRevision?.apiRevisionType === 'pullRequest') {
+              this.loadAssociatedPullRequestData(this.reviewId!, this.activeApiRevisionId!);
+            }
           }
         }
       });
   }
 
+  loadAssociatedPullRequestData(reviewId: string, activeApiRevisionId: string) {
+    this.apiRevisionsService.getAssociatedPullRequest(reviewId, activeApiRevisionId)
+    .subscribe(response => {
+      this.associatedPRs = response;
+      this.pullRequestData = response.map(pr => ({
```

Comment:
Since you are persisting the entire response in memory, `pullRequestData` is not needed.